content/it/contributor-ladder/_index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Livelli di Contributor
+toc_hide: true
+status: Completed
+menu:
+  main:
+    weight: 10
+---
+
+Benvenuto! 👋 Grazie per il tuo interesse nel voler contribuire al progetto del Glossario Cloud Native CNCF Cloud Native Glossary. 
+Ci sono molti modi per diventare parte attiva di questa comunità: 
+puoi contribuire con nuovi termini, o localizzare il Glossario nella tua lingua madre, o ancora aiutare gli altri ad iniziare. 
+Questo documento spiega i diversi **livelli nel percorso di contribuzione** (cioè la progressione in termini di esperienza e ingaggio all'interno del gruppo delle persone che collaborano al progetto come _Contributor_, appunto), assieme a oneri e onori che ne conseguono. 
+
+## 1. Contributor
+
+Il Glossario è per tuttɜ, e tuttɜ possono diventare _Contributor_ del Glossario semplicemente contribuendo al progetto.
```

Comment:
```suggestion
Chiunque può diventare _Contributor_ del Glossario semplicemente contribuendo al progetto.
```

Reply:
Bello! Si perde un po' il primo concetto, ma in effetti non è fondamentale in questa pagina.